hooks/pre-commit

```diff
@@ -23,7 +23,7 @@ chronic typos .
 (cd relayer && chronic mage lint && chronic go fmt ./...)
 
 # format rust
-(cd parachain && chronic cargo +nightly-"$SNOWBRIDGE_RUST_NIGHTLY" fmt)
+(cd parachain && chronic cargo +nightly-"$SNOWBRIDGE_RUST_NIGHTLY" fmt --check)
```

Comment:
Seems there are still several unformatted codes submitted to the repo, so force an `fmt check` here and require manually run `cargo +nightly fmt` for the fix.